modules/nf-core/cowpy/tests/nextflow.config

```
@@ -0,0 +1,5 @@
+process {
+    withName: COWPY {
+        ext.args = {"${params.cowpy_args}"}
```

Interesting. We normally just use this: `ext.args = params.module_args`
```suggestion
        ext.args = {"${params.module_args}"}
```
but then you have to define the params for all tests, whether you want to specify any extra args or not. Whereas this construction will always evaluate to something (potentially an empty string).